app/views/_includes/applications/gcses.njk

```diff
@@ -1,5 +1,5 @@
-<h2 class="govuk-heading-m">GCSEs or equivalent</h2>
-<div class="govuk-!-margin-bottom-8">
+<h3 class="govuk-heading-m govuk-!-margin-top-5">GCSEs or equivalent</h3>
```

Comment:
We could make future iterations easier if we specify a heading class rather than an amalgamation of overrides